Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -281,20 +287,22 @@ open class AVFoundationPlayback: Playback {
     }
 
     open override func seek(_ timeInterval: TimeInterval) {
-        if supportDVR {
-            var timeToSeek = timeInterval
-
-            if let seekStartTime = dvrWindowStart {
-                timeToSeek = timeToSeek + seekStartTime
-            }
+        seek(relativeTime(to: timeInterval)) { [weak self] in
+            self?.triggerDvrStatusIfNeeded()
+        }
+    }
 
-            seek(timeToSeek) { [weak self] in
-                if let usingDVR = self?.usingDVR {
-                    self?.trigger(.usingDVR, userInfo: ["enabled": usingDVR])
-                }
-            }
+    private func relativeTime(to time: TimeInterval) -> TimeInterval {
+        if isDvrAvailable {
+            return time + (dvrWindowStart ?? 0)
```

Comment:
We need to check these relative time calculations when there are discontinuities, just to have a predictable behavior.